dotcom-rendering/src/components/marketing/epics/ContributionsEpic.tsx

```diff
@@ -405,14 +405,13 @@ const ContributionsEpic: ReactComponent<EpicProps> = ({
 			{showAboveArticleCount && (
 				<div css={articleCountAboveContainerStyles}>
 					<ContributionsEpicArticleCountAboveWithOptOut
-						articleCounts={articleCounts}
+						articleCount={articleCounts.forTargetedWeeks}
 						isArticleCountOn={!hasOptedOut}
 						onArticleCountOptOut={onArticleCountOptOut}
 						onArticleCountOptIn={onArticleCountOptIn}
 						openCmp={openCmp}
 						submitComponentEvent={submitComponentEvent}
 						copy={variant.separateArticleCount?.copy}
-						countType={variant.separateArticleCount?.countType}
```

Comment:
this `countType` isn't configured anywhere, so we always end up defaulting to the `for52Weeks` count

Reply:
the alternative is to actually make `countType` configurable from the RRCP, but I don't think that's necessary currently.

Reply:
Ha I missed this when asking this same question below :-)